speechbrain/nnet/quaternion_networks/q_CNN.py

```diff
@@ -189,6 +195,21 @@ def forward(self, x):
         """
         # (batch, channel, time)
         x = x.transpose(1, -1)
+
+        if self.max_norm is not None:
```

Comment:
is renorming individual quaternion components actually is strictly equivalent to renorming the quaternion?

Reply:
No I don't believe it is strictly equivalent. I couldn't find any references for how to approach it so I went with the simplest idea which was the component-wise renorm.